src/backends/aws/AmazonS3Storage.php

```diff
@@ -11,13 +11,13 @@ public function __construct($options) {
 		}
 		$bucket = $options['bucket'];
 	  	parent::__construct('s3://' . $bucket);
-	  	$this->aws = new Aws\S3\S3Client(array(
+
```

Comment:
Using Aws\Common\Credentials class is proper way and also promoted by Aws. Hence it has been picked up over both implementation.